plugin-server/src/main/utils.ts

```diff
@@ -101,6 +101,8 @@ export async function setupKafkaHealthcheckConsumer(kafka: Kafka): Promise<Consu
     const consumer = kafka.consumer({
         groupId: `${KAFKA_PREFIX}healthcheck-group`,
         maxWaitTimeInMs: 100,
+        sessionTimeout: 15000,
```

Comment:
Can we also increase the timeout to be 15s + 30s + some delta?

Reply:
We discussed this on Zoom and the TLDR is that we should increase the overall `/_health` [endpoint timeout](https://github.com/PostHog/posthog/blob/9a33696351ee63c0dbc6afacd81fea919e4a6b9f/plugin-server/src/main/services/http-server.ts#L21) (default 20 second) as per the worst case, it will take 15 min to detect group leader failure + max 30 second for the group rebalancing.

We decided to not push this as well to don't change too many things at once. We'll leave this for #team-ingestion